src/Blog/View/posts.twig

```diff
@@ -1,25 +1,71 @@
 {% set posts = [
     {
         'id': '1',
-        'title': 'Lorem ipsum',
+        'title': 'Jak efektywnie uczyć się programowania?',
         'preamble': 'Lorem ipsum dolor sit amet, consectetur adipiscing elit.',
         'url':'/img/slider1.svg',
         'date':'10-02-2020'
     }
-] %}
+	] %}
 {% include 'Common/View/header.twig' %}
 {% block body %}
-	<body class="top-wave">
+	<body class="top-wave" style="font-family:Lato;">
 	{% endblock %}
 	<section class="container">
-		<h1>Blog</h1>
 		<ul>
-			{% for post in posts %}
-				<li>
-					<a href="/blog/{{ post.id }}">{{ post.title }}</a>
-				</li>
-			{% endfor %}
-		</ul>
+		<select>
+		<option selected="">Sortuj</option>
+		<option>Od A do Z</option>
+		<option>Od Z do A</option>
+		<option>Najświeższe</option>
+		<option>Najstarsze</option>
+		</select>
+		{% for post in posts %}
+			<li class="u-margin--medium clearfix">
```

Comment:
Well done with HTML :) no errors in semantic structure, in class names and no omissions or mistakes :)